test/functional/specs/actions/sendEvent.spec.js

```diff
@@ -343,7 +343,7 @@ test("returns minimal valid settings", async () => {
   });
 
   await extensionViewController.expectIsValid();
-  await extensionViewController.expectSettings({
+  await extensionViewController.expectSettingsToContain({
```

Comment:
The view is instantiated with a number of instances (alloy1, alloy2). When you get the settings, the object contains more properties than the instance name. I don't know how was this test passing before.